bookwyrm/templates/guided_tour/book.html

```diff
@@ -0,0 +1,160 @@
+{% load i18n %}
+
+<script>
+    const tour = new Shepherd.Tour({
+        exitOnEsc: true,
+    });
+
+    tour.addSteps([
+        {
+            text: "{% trans 'This is home page of a book. Let\s see what you can do while you\'re here!' %}",
+            title: "{% trans 'Book page' %}",
+            buttons: [
+                {
+                    action() {
+                        disableGuidedTour(csrf_token);
+                        return this.complete();
+                    },
+                    secondary: true,
+                    text: "{% trans 'End Tour' %}",
+                    classes: "is-danger",
+                },
+                {
+                    action() {
+                        return this.next();
+                    },
+                    text: "{% trans 'Next' %}",
+                },
+            ],
+        },
+        {
+            text: "{% trans 'This is where you can set a reading status for this book. You can press the button to move to the next stage, or use the drop down button to select the reading status you want to set.' %}",
+            title: "{% trans 'Reading status' %}",
+            attachTo: {
+                element: "#tour-shelve-button",
+                on: "right",
+            },
+            buttons: [
+                {
+                    action() {
+                        return this.back();
+                    },
+                    secondary: true,
+                    text: "{% trans 'Back' %}",
+                },
+                {
+                    action() {
+                        return this.next();
+                    },
+                    text: "{% trans 'Next' %}",
+                },
+            ],
+        },
+        {
+            text: "{% trans 'You can also manually add reading dates here. Unlike changing the reading status using the previous method, adding read-through dates manually will not automatically add them to your ' %}\
+            <code>\
+            {% trans 'Read' %}\
+            </code>\
+            {% trans ' or '%}\
+            <code>\
+            {% trans 'Reading' %}\
+            </code>\
+            {% trans ' shelves. Got a favourite you re-read every year? We\'ve got you covered - you can add multiple read dates for the same book 😀' %}",
```

Comment:
Breaking up the translation string will make this hard to translate. It would work better to include it in one long block or break it up by sentence. It's better to include the html in the string rather than end the translation block to avoid it.

If you want to keep the whitespace for readability, you can use `{% blocktrans trimmed %}...{% endblocktrans %}` instead of `{% trans "..." %}`.